aries_cloudagent/vc_api/routes.py

```diff
@@ -0,0 +1,227 @@
+"""VC-API Routes."""
+
+from aiohttp import web
+from aiohttp_apispec import docs, request_schema, response_schema
+
+from ..admin.request_context import AdminRequestContext
+from ..config.base import InjectionError
+from ..resolver.base import ResolverError
+from ..wallet.error import WalletError
+
+from ..vc.ld_proofs.document_loader import DocumentLoader
+from ..vc.ld_proofs.purposes.assertion_proof_purpose import AssertionProofPurpose
+
+from ..vc.vc_ld.verify import verify_credential, verify_presentation
+from ..vc.vc_ld.issue import issue as issue_credential
+from ..vc.vc_ld.prove import sign_presentation
+from ..vc.vc_ld.store import store_credential
+from ..vc.vc_ld.manager import VcLdpManager, VcLdpManagerError
+from ..vc.vc_ld.models.credential import VerifiableCredential
+from ..vc.vc_ld.models.presentation import VerifiablePresentation
+from ..vc.vc_ld.models.options import LDProofVCOptions
+
+from .examples import (
+    IssueCredentialRequest,
+    IssueCredentialResponse,
+    VerifyCredentialRequest,
+    VerifyCredentialResponse,
+    ProvePresentationRequest,
+    ProvePresentationResponse,
+    VerifyPresentationRequest,
+    VerifyPresentationResponse,
+)
```

Comment:
I'm happy with using the current models in the vc module, these quick examples were more to have a usable swagger interface to demonstrate this easily. I would remove the examples file and use the existing vc models to avoid replicating code.

Reply:
Oh, I may have misinterpreted what these represented, woops. Sounds good.